### DISCUSSION TITLE
Update the nightly build?

### DISCUSSION THREAD

**Discussion by hnaether** [HASwitchPlate/openHASP#762] [General]
Hi everyone, 
thank you for this fantastic project. I was wondering why the latest nightly build is from may? Should it be triggered automatically every night - or is this a manual process I can manually execute?
Thanks ,
Hans

**Comment by FreeBear-nc** [comment]
It is possible to configure github to automatically generate nightly builds. But it is not a trivial task, and only fvanroie as the repository owner can set this up.

I've looked at the github docs, and it scares me ;-0

**Reply by fvanroie** [reply]
Githib Actions are already setup and running: https://github.com/HASwitchPlate/openHASP/actions

The website needs to be updated manually though...

**Reply by hnaether** [reply]
Ok, thank you for your explanation. Clear for me now. I can download the images from https://github.com/HASwitchPlate/openHASP/actions

Cheers
Hans
